### PR TITLE
Fix some deprecations (shadowing / body)

### PR DESCRIPTION
**Pull request by Geod24**
Pretty straightforward.

### CODE REVIEW

**Comment by Geod24** [comment]
Looks like a spurious failure ?

File: dstep/translator/objc/ObjcInterface.d [hunk 1]
```diff
@@ -31,10 +31,10 @@ class ObjcInterface (Data) : Declaration
 
     override void translate (Output output)
     {
-        auto cursor = cursor.objc;
+        auto out_cursor = cursor.objc;
```
Comment (jacob-carlborg) [inline dstep/translator/objc/ObjcInterface.d]: Please use camel casing.

File: dstep/translator/objc/ObjcInterface.d [hunk 2]
```diff
@@ -64,11 +64,11 @@ class ObjcInterface (Data) : Declaration
         output.output(currentClass.data());
     }
 
-    protected string[] collectInterfaces (ObjcCursor cursor)
+    protected string[] collectInterfaces (ObjcCursor obj_cursor)
```
Comment (jacob-carlborg) [inline dstep/translator/objc/ObjcInterface.d]: Same here, camel casing. And perhaps name it `objcCursor`. “objc” is the standard abbreviation of Objective-C.

**Comment by jacob-carlborg** [comment]
> Looks like a spurious failure ?

Most likely yes.

**Comment by Geod24** [comment]
Addressed comments but the failure persists.

**Comment by jacob-carlborg** [comment]
Thanks.

**Pull request merged**